poetry.lock

```diff
@@ -4422,20 +4447,6 @@ files = [
 [package.dependencies]
 types-urllib3 = "*"
 
-[[package]]
-name = "types-requests"
```

Comment:
Weird that this would be removed. It's listed as a dev dependency in `pyproject.toml`, so shouldn't it be here? I don't know anything about Poetry.